internal/prep/prep.go

```diff
@@ -246,10 +246,14 @@ func SetupStateroot(log logr.Logger, ops ops.Ops, ostreeClient ostreeclient.ICli
 		return fmt.Errorf("failed to process etc.deletions: %w", err)
 	}
 
-	if err := common.CopyOutsideChroot(filepath.Join(mountpoint, "containers.list"), imageListFile); err != nil {
+	if err := common.CopyOutsideChroot(filepath.Join(mountpoint, "containers.list"), filepath.Join(tmpPath, "containers.list")); err != nil {
```

Comment:
don't we have var in common for containers.list? Lets add it in this case

Reply:
Took your suggestion and did a small refactoring into how we manage common filepaths between IBu/IBI. PTAL :pray: